Gltf/GltfGenerator/GltfGenerator.csproj

```
@@ -1,22 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <LangVersion>preview</LangVersion>
-    <Version>2.0.0</Version>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
+  
+  <!-- Package references needed for source generators -->
+  <ItemGroup>
```

```suggestion
  <ItemGroup Label="Package references needed for source generators">
```
Take it or leave it, just a minor suggestion I only just learnt myself while testing & reviewing this (;